optimum/exporters/openvino/model_configs.py

```diff
@@ -90,6 +91,21 @@ def init_model_configs():
 register_in_tasks_manager = TasksManager.create_register("openvino", overwrite_existing=True)
 
 
+@register_in_tasks_manager(
```

Comment:
I do not see any difference with onnx version config. Why do you need to have it explicitly?

Reply:
Added as `DEFAULT_ONNX_OPSET` now needs to be set to 14 (now that F.scaled_dot_product_attention is used by default for this model), will be removed before merging the PR as it'll be added in optimum directly